### ISSUE TITLE
[Feature request] Archive

### ISSUE THREAD

**Issue by inson1** [dstark5/Openlib#67]
It would be great for managing books, which I already read or which just work as now some downloaded books dont work

Option to add/remove it from Archive
2 options in My Library in header- My Library, Archive

**Comment by dstark5** [comment]
Yeah that's a great idea will sure add it in the upcoming updates
